### PR TITLE
fix(RHINENG-9245): Fix API doc permissions for staleness

### PR DESCRIPTION
**Pull request by jpramos123**
# Overview

This PR is being created to address [RHINENG-9245](https://issues.redhat.com/browse/RHINENG-9245).

Fix API doc with correct permissions for the staleness resources

## PR Checklist

- [ ] Keep PR title short, ideally under 72 characters
- [ ] Descriptive comments provided in complex code blocks
- [ ] Include raw query examples in the PR description, if adding/modifying SQL query
- [ ] Tests: validate optimal/expected output
- [ ] Tests: validate exceptions and failure scenarios
- [ ] Tests: edge cases
- [ ] Recovers or fails gracefully during potential resource outages (e.g. DB, Kafka)
- [ ] Uses [type hinting](https://docs.python.org/3/library/typing.html), if convenient
- [ ] Documentation, if this PR changes the way other services interact with host inventory
- [ ] Links to related PRs

## Secure Coding Practices Documentation Reference

You can find documentation on this checklist [here](https://github.com/RedHatInsights/secure-coding-checklist).

## Secure Coding Checklist

- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

**Review by strider — APPROVED** [review]
LGTM

File: swagger/api.spec.yaml [hunk 1]
```diff
@@ -792,7 +792,7 @@ paths:
       summary: Read the entire list of account staleness
       description: >-
         Read the entire list of all accounts staleness available.
-        Required permissions: inventory:TODO:read
+        Required permissions: staleness:staleness:read
```
Comment (thearifismail) [inline swagger/api.spec.yaml:795]: What are the firs and second staleness in `staleness:staleness:read/write`?  Should these names be changed slightly for easy differentiation?

Comment (jpramos123) [reply swagger/api.spec.yaml:795]: @thearifismail This naming convention was aligned at the begging of the project, and they are already running in production... I think we are good if we stick to this

Comment (thearifismail) [reply swagger/api.spec.yaml:795]: okay.  What do the first and second staleness mean?

**Review by thearifismail — APPROVED** [review]
Please respond to my question if you have time.

**Pull request merged**